rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,150 @@
+
+### **User Stories**
+
+1.  > As a researcher new to the DCP, I would like to get a timely
+    > answer to my question so that I can have a successful experience
+    > submitting, exploring, downloading, or analyzing HCA data.
+
+2.  > As a DCP PM, PO or PL, I would like to be assured that Zendesk
+    > tickets are being responded to in a timely manner so that I can
+    > focus on my own tasks without having to micromanage responses.
+
+3.  > As a DCP team member, I would like to know whether or not I should
+    > be responding to a Zendesk ticket so that I can either respond in
+    > a timely manner or be assured that someone else is responding in a
+    > timely manner.
+
+## Scientific "guardrails" \[optional\]
+
+
+## **Detailed Design**
+
+This section describes the flow of an issue through the support state
+diagram presented below.
+
+![](../images/0000-Zendesk-triage-SOP-diagram.jpeg)
+
+## Issue Creation
+
+Issues in Zendesk originate from users of the DCP. These users could
+include data contributors, researchers trying to use the data in the
+DCP, software developers trying to leverage the DCP API. Any user of the
+DCP can file an issue. 
+
+While problems identified by an internal DCP staff member could be filed on Zendesk too, we favor a plan in which those issues are directly added to the applicable development
+repository, where they can be addressed directly. This plan eliminates the extra work for the triager of managing internally-generated tickets.
+
+Once an issue is
+created in Zendesk there is a tiered support network that kicks in to
+resolve the question.
+
+## Primary Support
+
+The triage person, or triage team, is responsible for receiving initial
+support requests from end users, and monitors the Zendesk system daily
+for new tickets. Note that the Zendesk system automatically sends out an initial canned response to the user, indicating that the request was received and is being reviewed by our support staff. 
+
+The Service Level Objective is to respond to the user with a solution no later than 48 business hours after it is filed.
+
+The triager takes ownership of each open ticket and does the following to develop a response:
+
+1. Evaluate whether there is enough information in the ticket to inform the
+response. If not, write back to the user and ask for the missing information. 
+
+2. Search the data portal documentation and FAQ for answers to the question. If the triage person can find an answer that resolves the problem, a repsonse is sent to the user and the issue is closed. 
+
+3. If an answer is not evident from searching the knowledge base in the data portal, the triager needs to ask for help from the secondary support team, which includes members from each DCP component or service. It may
+be obvious which team should receive the ticket; if so, then it can be
+assigned directly. If it’s not obvious, there can be some internal
+discussion with the secondary support staff about who should receive the
```

Comment:
Bit _hand wavy_ - for example, there's no mention of the zendesk slack channel? What happens if secondary support ignores the request because they're busy shipping code?

Reply:
I took a shot at resolving this.